src/bin/ion/commands/beta/count.rs

```diff
@@ -1,25 +1,25 @@
-use crate::commands::CommandConfig;
 use anyhow::{Context, Result};
-use clap::{App, Arg, ArgMatches};
+use clap::{Arg, ArgAction, ArgMatches, Command};
 use ion_rs::*;
 use std::fs::File;
 use std::io::{stdin, BufReader, StdinLock};
 
-pub fn app() -> CommandConfig {
-    App::new("count")
+pub fn app() -> Command {
+    Command::new("count")
         .about("Prints the number of top-level values found in the input stream.")
         .arg(
             // All argv entries after the program name (argv[0])
             // and any `clap`-managed options are considered input files.
-            Arg::with_name("input")
+            Arg::new("input")
                 .index(1)
-                .multiple(true)
-                .help("Input file [default: STDIN]"),
+                .help("Input file [default: STDIN]")
+                .action(ArgAction::Append)
+                .trailing_var_arg(true),
```

Comment:
🗺️ The `multiple(true)` setting has been removed. You can get the same effect by changing the `ArgAction` associated with the arg name from `Set` to `Append`.

Setting `trailing_var_arg` to `true` indicates that all trailing arguments not associated with a flag will be part of `input`.